Dockerfile

```diff
@@ -1,22 +1,61 @@
-FROM ubuntu:bionic
+# syntax=docker/dockerfile:experimental
 
-# System dependencies
-RUN apt-get update && apt-get install -y --no-install-recommends python3 python3-setuptools python3-pip
+# Build stage: Install python dependencies
+# ===
+FROM ubuntu:bionic AS python-dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+ADD requirements.txt /tmp/requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement /tmp/requirements.txt
+
+
+# Build stage: Install yarn dependencies
+# ===
+FROM node:10-slim AS yarn-dependencies
```

Comment:
Current LTS is the version 12 line

Reply:
Oh, and more importantly we use 12 in the run script - https://github.com/canonical-web-and-design/docker-dev/blob/master/Dockerfile#L10. I thought we used 10, but I should have checked.

Reply:
https://github.com/canonical-web-and-design/jaas.ai/pull/496